cmds.go

```diff
@@ -44,3 +56,65 @@ func (sc *SonyCmdPower) Run(cli *CLI) error {
 	}
 	return bc.SetPowerStatus(status)
 }
+
+// Run (sony input) gets or sets the currently displayed input of a Sony Bravia
+// TV set. If no argument is provided and the flag --list is not specified, the
+// currently selected input is printed with the label of the input as
+// configured on the TV, or with an input URI if no label is set. If --list is
+// specified, all the available input URIs with their labels (if any) are
+// listed. If an argument is provided and matches the label of one of the
+// inputs, the TV is set to that input. Otherwise the argument is assumed to be
+// a URI and sets the input to that URI.
+func (sc *SonyCmdInput) Run(cli *CLI) error {
+	bc := NewRESTClient(cli.TV.Hostname, cli.TV.PSK)
+	labels, err := bc.Inputs()
+	if err != nil {
+		return fmt.Errorf("getting labels: %w", err)
+	}
+
+	switch {
+	// List all inputs
+	case sc.Label == "" && sc.List:
+		tw := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
+		fmt.Fprintln(tw, "URI\tLABEL")
+
+		// Get the URIs from the map and sort them
+		uris := make([]string, 0, len(labels))
+		for k := range labels {
+			if strings.HasPrefix(k, "extInput:") {
+				uris = append(uris, k)
+			}
+		}
+		sort.Strings(uris)
+
+		for _, uri := range uris {
+			fmt.Fprintf(tw, "%s\t%s\n", uri, labels[uri])
+		}
+		tw.Flush() //nolint:errcheck,gosec
+
+	// Show selected input
+	case sc.Label == "" && !sc.List:
+		uri, err := bc.SelectedInput()
+		if err != nil {
+			return fmt.Errorf("selected input: %w", err)
+		}
+		label := labels[uri]
+		if label == "" {
+			label = "unlabelled: " + uri
+		}
+		fmt.Println(label)
+
+	// Select input by label
+	case sc.Label != "" && !sc.List:
+		uri := labels[sc.Label]
+		if uri == "" {
+			uri = sc.Label
+		}
+		if err := bc.SetInput(uri); err != nil {
+			return fmt.Errorf("set input: %w", err)
+		}
+	case sc.Label != "" && sc.List:
+		return errors.New("cannot use --list with a label") //nolint:goerr113 // dont care
```

Comment:
"dont care" is not very meaningful ;)

Reply:
dont care

Reply:
:tongue:

I mean, I don't care about that lint error. It does not need its own sentinel error. Perhaps if there was some sort of kong "UsageError" I would use that.

Reply:
Fine. Fixed in 975ddc71aa47f2e07123852d103ba1c42027adbc